src/cpp/rtps/security/SecurityManager.cpp

```diff
@@ -123,12 +124,95 @@ bool SecurityManager::init(
     security_activated = false;
 
     SecurityException exception;
+
     domain_id_ = participant_->getRTPSParticipantAttributes().builtin.domainId;
 
+    const PropertyPolicy log_properties = PropertyPolicyHelper::get_properties_with_prefix(
+          participant_->getRTPSParticipantAttributes().properties,
+          "dds.sec.log.builtin.DDS_LogTopic.");
+
+    // length(log_properties) == 0 considered as logging disable.
+    if (PropertyPolicyHelper::length(log_properties) > 0)
+    {
+      logging_plugin_ = factory_.create_logging_plugin(participant_properties);
+
+      if (logging_plugin_ != nullptr)
+      {
+        LogOptions log_options;
+        log_options.distribute = true; //TODO(artivis): defaults to true until log to file is implemented.
```

Comment:
Log to file is not implemented, so this can probably be changed now,  yes?

Reply:
Indeed.